src/Sentry/Platforms/Cocoa/SentryOptions.cs

```diff
@@ -215,4 +217,38 @@ public void AddInAppInclude(string prefix)
             InAppIncludes.Add(prefix);
         }
     }
+
+    // We actually add the profiling integration automatically in InitSentryCocoaSdk().
+    // However, if user calls AddProfilingIntegration() multiple times, we print a warning, as usual.
+    private bool _profilingIntegrationAddedByUser = false;
+
+    /// <summary>
+    /// Adds ProfilingIntegration to Sentry.
+    /// </summary>
+    /// <param name="startupTimeout">
+    /// Unused, only here so that the signature is the same as AddProfilingIntegration() from package Sentry.Profiling.
+    /// </param>
+    public void AddProfilingIntegration(TimeSpan startupTimeout = default)
+    {
+        if (HasIntegration<ProfilingIntegration>())
+        {
+            if (_profilingIntegrationAddedByUser)
+            {
+                DiagnosticLogger?.LogWarning($"{nameof(ProfilingIntegration)} has already been added. The second call to {nameof(AddProfilingIntegration)} will be ignored.");
```

Comment:
Wondering if a warning is needed. Because if I have a shared Init logic for Sentry by used Apps, I just want profiling to be there. the fact iOS has it automatically but not the other platforms is irrelevant.
```suggestion
                DiagnosticLogger?.LogDebug($"{nameof(ProfilingIntegration)} on iOS is added automatically. The second call to {nameof(AddProfilingIntegration)} will be ignored.");
```

Reply:
I've added the warning (actually the exact line of code) from the other profiling integration. Arguably, it may be unnecessary to warn when you add the same thing multiple times, no clue why integrations have warnings about that.

> Because if I have a shared Init logic for Sentry by used Apps, I just want profiling to be there. the fact iOS has it automatically but not the other platforms is irrelevant.

Doesn't play a role here. The warning only triggers when a user added the integration **manually** multiple times. Not when they add it once after we've added it automatically during init.

I'll go ahead and merge this as is so it's exactly the same as the other profiling integration (which also warns only when you add it manually multiple times). We can change it later in all the places with he same LogWarning if you think we should